cats/optimise_starttime.py

```diff
@@ -24,7 +28,11 @@ def get_starttime(data, method="simple", duration=None):
 
     if method == "windowed":
         # get length of interval between timestamps
-        interval = (data[1].datetime - data[0].datetime).total_seconds() / 60
-        # count number of intervals in size
-        num_intervals = int((duration / interval) + 0.5)  # round to nearest integer (UP)
-        return min(WindowedForecast(data, num_intervals))
+        interval = (
+            data[1].datetime - data[0].datetime
+        ).total_seconds() / 60
+        wf = WindowedForecast(
+            data=data,
+            window_size=ceil(duration / interval)
+        )
+        return wf[0], min(wf)
```

Comment:
As it stands, the function returns one argument if `method=="simple"` and two otherwise, which is not ideal. We could simply remove "simple" as it doesn't make sense from a CO2-minimisation point of view anyway?

Reply:
Agreed, I'd be happy to remove it. But let's adress that in another PR maybe.